code/game/objects/items/stacks/medical.dm

```diff
@@ -75,25 +77,24 @@
 			user.visible_message("<span class='green'>[user] applies [src] on [t_himself].</span>", "<span class='green'>You apply [src] on yourself.</span>")
 
 
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		affecting = H.get_bodypart(check_zone(user.zone_selected))
+	if(iscarbon(M))
+		var/mob/living/carbon/C = M
+		affecting = C.get_bodypart(check_zone(user.zone_selected))
 		if(!affecting) //Missing limb?
-			user << "<span class='warning'>[H] doesn't have \a [parse_zone(user.zone_selected)]!</span>"
+			user << "<span class='warning'>[C] doesn't have \a [parse_zone(user.zone_selected)]!</span>"
 			return
-		if(stop_bleeding)
-			if(!H.bleedsuppress) //so you can't stack bleed suppression
-				H.suppress_bloodloss(stop_bleeding)
-		if(affecting.status == ORGAN_ORGANIC) //Limb must be organic to be healed - RR
-			if(affecting.heal_damage(src.heal_brute, src.heal_burn, 0))
-				H.update_damage_overlays(0)
-
-			M.updatehealth()
+		if(ishuman(C))
+			var/mob/living/carbon/human/H = C
+			if(stop_bleeding)
+				if(!H.bleedsuppress) //so you can't stack bleed suppression
+					H.suppress_bloodloss(stop_bleeding)
+		if(affecting.status == BODYPART_ORGANIC) //Limb must be organic to be healed - RR
+			if(affecting.heal_damage(heal_brute, heal_burn))
+				C.update_damage_overlays()

```

Comment:
medical packs now heal the limb targeted by the user when applied on monkeys and alien, no just humans.